src/Contracts/DataModels.ts

```diff
@@ -433,7 +433,7 @@ export interface NotebookWorkspaceConnectionInfo {
 
 export interface ContainerInfo {
   durationLeftInMinutes: number;
-  notebookServerInfo: NotebookWorkspaceConnectionInfo;
```

Comment:
similar to controlplane changes, should we not have both versions here?
If this gets deployed to MPAC and we dont have the changes on stage, it will break.